Jim dev

jsharples777 opened the pull request:
1.  Directory structure
2.  Starting files
3. Server to run the API call

jsharples777 committed:
Initial

jsharples777 committed:
First commit
1.  Directory file structure
2.  NPM modules and start script to run the web server
3.  .ENV file (used by Node.js) to store key EDAMAM values
4.  FetchUtil - for executing POST fetches returning JSON with a callback
5.  Simple logging utility
6.  Controller class outline - for client-side code execution
7.  Basic index.html including Bulma css (installed locally)
8.  The details of the recipe search API (JSON format)
9.  App.css (empty) for the application CSS (referenced in index.html)

Need to run (if have the npm tool installed)

npm install

This will install all the modules for the client and server

TODO:
Start working on the server-side api call

raven-bootcamp commented on package.json:
```diff
@@ -0,0 +1,32 @@
+    "url": "https://github.com/jsharples777/template-feo-react-babel/issues"
+  },
+  "homepage": "https://github.com/jsharples777/template-feo-react-babel#readme"
+}
```
You'll probably have to take us through a few of these Jim, as you mentioned the other night

raven-bootcamp reviewed (approved):
Looks good, there's a few little things I haven't learned about yet, but I'm sure that will come in time. 👍

AMIrene reviewed (approved):
thanks jim!

jsharples777 committed:
1.  Added API call on server side
2.  Added call function on Controller with callback for the results
3.  Added test call function on Controller
4.  Added call of test function in App.js
5.  Corrected package.json to reference the correct Git Repo

Server-side API call is working, and tested
